diff --git a/node_modules/selenium-webdriver/lib/webdriver.js b/node_modules/selenium-webdriver/lib/webdriver.js
index b8d7bca..a5c7c6f 100755
--- a/node_modules/selenium-webdriver/lib/webdriver.js
+++ b/node_modules/selenium-webdriver/lib/webdriver.js
@@ -719,6 +719,13 @@ class WebDriver {
   static createSession(executor, capabilities, onQuit = undefined) {
     let cmd = new command.Command(command.Name.NEW_SESSION)
     
+    // The line of code below was removed in versions >4.4.0. This causes our
+    // BrowserStack tests to fail. Without this line BrowserStack cannot open
+    // the dev URL, which causes network errors.
+    //
+    // For OSS remote ends.
+    cmd.setParameter('desiredCapabilities', capabilities)
+
     // For W3C remote ends.
     cmd.setParameter('capabilities', {
       firstMatch: [{}],
